hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkDataSetProcessor.java

```diff
@@ -763,7 +784,14 @@ private static Column createFilterCondition(Dataset dataset,String[] allColIdInS
                 if (q.getVariantType() == Qualifier.VARIANT)
                     continue; // Cannot Push Down Qualifier
                 q.clearOrderableCache();
-                Column orCol = dataset.col(allColIdInSpark[(baseColumnMap != null ? baseColumnMap[q.getStoragePosition()] : q.getStoragePosition())]);
+//                Column orCol = dataset.col(allColIdInSpark[(baseColumnMap != null ? baseColumnMap[q.getStoragePosition()] : q.getStoragePosition())]);
+                Column orCol = dataset.col(allColIdInSpark[q.getStoragePosition()]);
```

Comment:
It's a big change to remove usage of baseColumnMap, though it looks like this is correct.  We might want to go through all types of external table paths and verify this is correct for all paths.

Reply:
Why is it correct? Is baseColumMap not needed here?

Reply:
As per my understanding, the qualifier q was already computed correctly, using index in the base dataset. If we use baseColumnMap, then we should do another reverse lookup to make it correct, which is redundant. In the existing code for 'and' qualifier, it is also the case that baseColumnMap is not used.